rustfmt.toml

```diff
@@ -1,9 +1,11 @@
 version = "Two"
 comment_width = 120
+edition = "2021"
```

Comment:
Is this necessary?

Reply:
It's not mandatory for most cases, only when not going through cargo fmt itself and I believe that's the case for on-save fmt on most IDEs. Doesn't hurt to have it anyway.